lib/commands/file-movement.js

```diff
@@ -148,35 +148,41 @@ async function pushFileToRealDevice (device, remotePath, base64Data) {
 }
 
 /**
- * Get the content of given file from iOS Simulator and return it as base-64 encoded string.
+ * Get the content of given file or folder from iOS Simulator and return it as base-64 encoded string.
+ * Folder content is recursively packed into a zip archive.
  *
+ * @param {boolean} isFile - Whether the destination item is a file or a folder
  * @param {Object} device - The device object, which represents the device under test.
  *                          This object is expected to have the `udid` property containing the
  *                          valid device ID.
- * @param {string} remotePath - The path to a file, which exists in the corresponding application
+ * @param {string} remotePath - The path to a file or a folder, which exists in the corresponding application
  *                              container on Simulator. The expected format of this string:
- *                              @<app_bundle_id>/<path_to_the_file_inside_container>
+ *                              @<app_bundle_id>/<path_to_the_file_or_folder_inside_container>
  * @returns {string} Base-64 encoded content of the file.
  */
-async function pullFileFromSimulator (device, remotePath) {
+async function pullFromSimulator (isFile, device, remotePath) {
```

Comment:
In terms of readable signatures, I would prefer `isFile` to be at the end rather than the beginning of the argument list. There it could also be defaulted.

Reply:
moved the argument to the last place